palvelutarjotin/urls.py

```diff
@@ -35,7 +37,14 @@ def healthz(*args, **kwargs):
 
 
 def readiness(*args, **kwargs):
-    return HttpResponse(status=200)
+    response_json = {
+        "status": "ok",
+        "release": settings.APP_RELEASE,
+        "packageVersion": __version__,
+        "commitHash": settings.REVISION.decode("utf-8"),
```

Comment:
https://kultus-api-pr321.dev.hel.ninja/readiness →
```json
{
    "status": "ok",
    "release": "",
    "packageVersion": "1.8.1",
    "commitHash": "579d7a6",
    "buildTime": "2023-07-19T11:44:56.000Z"
}
```
the commitHash is a [short](https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt---shortlength) version, thought at first that it would be good to have the whole commit hash for ease-of-use but tried it out and github found the same commit using both the short form and long form hashes:
- `https://github.com/City-of-Helsinki/palvelutarjotin/commit/579d7a668f229ce52e97fef9da7d776e24fa5544`
- `https://github.com/City-of-Helsinki/palvelutarjotin/commit/579d7a6`

so maybe it's not really needed.

Reply:
Added COMMITHASH that includes full hash. 

url is moved to 'api/version' that it will be same on all applications